README.md

```diff
@@ -71,10 +71,10 @@ The template can be as the following example:
 After the initial setup every time a change is made, the developer should run the following command in the project root path:
 
 ``` bash
-$ codelog new
+$ codelog new NAME
```

Comment:
The following command example shows `codelog release {x.y.z}`, what about `codelog new {name}` to make it more consistent?

Reply:
In the `release` command example, we omitted the optional parameter `DATE`. I suggest that we write that exactly like the output when the help command is called.

Reply:
We should add the `DATE` parameter to the README, then